pages/common/ssh-keygen.md

```diff
@@ -8,28 +8,28 @@
 
 - Specify file in which to save the key:
 
-`ssh-keygen -f ~/.ssh/{{filename}}`
+`ssh-keygen -f {{~/.ssh/filename}}`
 
 - Generate an ed25519 key with 100 key derivation function rounds:
 
-`ssh-keygen -t ed25519 -a 100`
+`ssh-keygen -t {{ed25519}} -a {{100}}`
 
 - Generate an RSA 4096 bit key with email as a comment:
 
-`ssh-keygen -t rsa -b 4096 -C "{{email}}"`
-
-- Retrieve the key fingerprint from a host (useful for confirming the authenticity of the host when first connecting to it via SSH):
-
-`ssh-keygen -l -F {{remote_host}}`
+`ssh-keygen -t {{dsa|ecdsa|ed25519|rsa}} -b {{4096}} -C "{{comment|email}}"`
```

Comment:
This example is not in the DE page.

Reply:
It is there, it's just missing all the other options. I'll make a suggestion for that